Update Community Cloud docs (#1149)
streamlit/docs: content/deploy/community-cloud/manage-your-app/_index.md

### App overflow menus

Each app has a menu accessible from the overflow icon (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>more_vert</i>) to the right.

- **Edit with Codespaces** &mdash; See [Edit your app with GitHub Codespaces](/deploy/streamlit-community-cloud/manage-your-app/edit-your-app#edit-your-app-with-github-codespaces)
- **Reboot** &mdash; See [Reboot your app](/deploy/streamlit-community-cloud/manage-your-app/reboot-your-app)
- **Delete** &mdash; See [Delete your app](/deploy/streamlit-community-cloud/manage-your-app/delete-your-app)
- **Analytics** &mdash; See [App analytics](/deploy/streamlit-community-cloud/manage-your-app/app-analytics)
- **Settings** &mdash; See [App settings](/deploy/streamlit-community-cloud/manage-your-app/app-settings)

![App overflow menu in your workspace](/images/streamlit-community-cloud/workspace-app-overflow.png)

If you have view-only access to an app, all options in the app's menu will be disabled except analytics.

![View-only app overflow menu in your workspace](/images/streamlit-community-cloud/workspace-view-only.png)

## Manage your app directly from your app

> You can manage your deployed app directly from the app itself! Just make sure you are signed in to Community Cloud then visit your app.

Reviewer: ```suggestion
You can manage your deployed app directly from the app itself! Just make sure you are signed in to Community Cloud, and then visit your app.
```

</Flex>

### App chrome

From your app at `<your-custom-subdomain>.streamlit.app`, you can always access the [app chrome](/develop/concepts/architecture/app-chrome) just like you can when developing locally. The option to deploy your app is removed, but you can still clear your cache from here.

![App menus in Streamlit Community Cloud](/images/streamlit-community-cloud/app-menu.png)

## Manage your app in GitHub

### Update your app

Your GitHub repository is the source for your app, so that means that any time you push an update to your repository you'll see it reflected in the app in almost real time. Try it out!

Streamlit also smartly detects whether you touched your dependencies, in which case it will automatically do a full redeploy for you—which will take a little more time. But since most updates don't involve dependency changes, you should usually see your app update in real time.

### Add or remove dependencies

> You can add or remove dependencies at any point by updating `requirements.txt` (Python dependenciess) or `packages.txt` (Linux dependencies) and committing the changes to your repository on GitHub. This will cause Community Cloud to detect there was a change in your dependencies and automatically trigger (re)installation.

Reviewer: ```suggestion
To add or remove dependencies at any point, just update `requirements.txt` (Python dependenciess) or `packages.txt` (Linux dependencies) and commit the changes to your repository on GitHub. Community Cloud detects the change in your dependencies and automatically triggers (re)installation.
```

### App hibernation

All apps without traffic for one weekday will go to sleep. The system checks apps for inactivity throughout each day as follows:

- Tuesday through Friday: All apps without traffic for 24 hours (one day) will go to sleep.
- Saturday through Monday: All apps without traffic for 72 hours (three days) will go to sleep.

> This is done to alleviate resources and allow the best communal use of the platform! If you would like to keep your app awake, simply visit the app or commit to your app's repository, even if it's an empty commit!

Reviewer: ```suggestion
This system conserves resources and allows the best communal use of the platform. To keep your app awake, simply visit the app or commit to your app's repository, even if it's an empty commit!
```
